### ISSUE TITLE
Miner frying PC

### ISSUE THREAD

**Issue by xzMan1337** [UnamSanctam/SilentCryptoMiner#282]
Hey Unam, 
I am experienceing a problem with the miner right now, for some reason when I run it with the ethash algorithm, even when I specify 0% GPU while not idling, the PC starts to lag extremly, sometimes to the point where I can't even shut it down the proper way, is there any program known running in the background doing this or it my config, my config is this: [https://anonfiles.com/b2Fckc97xc/test-config_xml](url)

**Comment by xzMan1337** [comment]
Hey I'm sorry for even opening this, it seems like every miner does this to my PC which is weird since I used other miners previously with no problem, might have a problem with my gpu :/

**Comment by UnamSanctam** [comment]
Yeah most likely some other program on your computer is using up all your VRAM.

**Comment by xzMan1337** [comment]
holy shit I have a VRAM usage of 63% maybe **I** have a miner running in the background haha, sorry for only checking this now, I thought I had made sure that the vram was empty guess that was just a glitch of afterburner
